website/docs/components/stepper/nav/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Stepper Nav
+description: >-
+  A visual indicator of an on-page linear progress, where users can see where they are in a given flow.
```

Comment:
```suggestion
  Navigates users through a sequential flow allowing them to visually track progress.
```

Reply:
I tried to clarify the description to make it easier for consumers to grasp the appropriate usage and distinguish from the Stepper List. (See related suggestions for editing the Stepper Nav description.)